backend/cmm_helpers.mli

```diff
@@ -1198,57 +1182,26 @@ val unboxed_int64_or_nativeint_array_set :
   expression
 
 (** {2 Getters and setters for unboxed int and float32 fields of mixed
-    blocks} *)
-
-(** The argument structure for getters is parallel to [get_field_computed]. *)
-
-val get_field_unboxed_int32 :
-  Asttypes.mutable_flag ->
-  block:expression ->
-  index:expression ->
-  Debuginfo.t ->
-  expression
-
-val get_field_unboxed_float32 :
-  Asttypes.mutable_flag ->
-  block:expression ->
-  index:expression ->
-  Debuginfo.t ->
-  expression
+    blocks} [immediate_or_pointer] is not needed as the layout is implied from the name,
+    and [initialization_or_assignment] is not needed as unboxed ints can always be
+    assigned without caml_modify (etc.). *)
```

Comment:
Even though i understand who you reached this point, it'as a bit weird to specify why these function do not take some arguments (all the more since you don't specify the arguments they do take).